README.md

```diff
@@ -21,19 +21,44 @@ INSTALLED_APPS = [
 
 ## Commands
 
-### initdb
+### psqlinit
 
-Django command initdb create a database defined in django settings. If postgres user is not allowed to create database you can define it via `--db-url` parameter (format from [django-environ](https://github.com/joke2k/django-environ)) 
+Django command psqlinit create a database defined in django settings. 
 
-If you are using some Postgres extenstions you can define it with `PSYCOPG2_EXTENSIONS` setting (list of extension names). Command will automatically create it.
+You can define specific database settings with standard django ``DATABASE`` settings:
 
-### cleandb
+```python
+DATABASES = {
+    'default': {
+        ...
+        'EXTENSIONS': ['postgis', 'unaccent'],  # extensions to be installed with psqlinit command
+        'SNAPSHOT_FILE': Path('data', 'sql', 'local', 'init_default.sql'), # SQL which will be loaded after database initialization
+    },
+}
+```
+
+Snapshot and extensions are automatically loaded when database is preparing for tests too.
+
+### psqlclean
 
-PostgreSQL database requires often call `VACUUM` and `REINDEX`. The command `cleandb` performs these operations. Again you can specify database or root user with `--db-url` parameter.
+PostgreSQL database requires often call `VACUUM` and `REINDEX`. The command `cleandb` performs these operations.
```

Comment:
The command `psqlclean` performs these operations.

Reply:
I would also mention that the command vacuums all tables if no arguments are provided.